packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -22,27 +22,122 @@
         </div>
       </div>
       <div class="overflow-x-auto">
-        <div
-          v-if="fetching && !error && usersList.length === 0"
-          class="flex justify-center"
-        >
+        <div v-if="fetching" class="flex justify-center">
           <HoppSmartSpinner />
         </div>
 
         <div v-else-if="error">{{ t('users.load_list_error') }}</div>
 
-        <UsersTable
-          v-else-if="usersList.length >= 1"
-          :usersList="usersList"
-          :fetching="fetching"
-          :error="error"
-          @goToUserDetails="goToUserDetails"
-          @makeUserAdmin="makeUserAdmin"
-          @makeAdminToUser="makeAdminToUser"
-          @deleteUser="deleteUser"
-        />
+        <HoppSmartTable v-else-if="usersList.length" :list="usersList">
+          <template #head>
+            <tr
+              class="text-secondary border-b border-dividerDark text-sm text-left bg-primaryLight"
+            >
+              <th class="px-6 py-2">{{ t('users.id') }}</th>
+              <th class="px-6 py-2">{{ t('users.name') }}</th>
+              <th class="px-6 py-2">{{ t('users.email') }}</th>
+              <th class="px-6 py-2">{{ t('users.date') }}</th>
+              <!-- Empty header for Action Button -->
+              <th class="px-6 py-2"></th>
+            </tr>
+          </template>
+
+          <template #body="{ list }">
+            <tr
+              v-for="user in list"
+              :key="user.uid"
+              class="text-secondaryDark hover:bg-divider hover:cursor-pointer rounded-xl"
+              @click="goToUserDetails(user.uid)"
+            >
+              <td class="py-2 px-7 max-w-30">
+                <div class="flex">
+                  <span class="truncate">
+                    {{ user.uid }}
+                  </span>
+                </div>
+              </td>
+
+              <td class="py-2 px-7">
+                <div class="flex items-center space-x-2">
+                  <span>
+                    {{ user.displayName ?? t('users.unnamed') }}
+                  </span>
+
+                  <span
+                    v-if="user.isAdmin"
+                    class="text-xs font-medium px-3 py-0.5 rounded-full bg-green-900 text-green-300"
+                  >
+                    {{ t('users.admin') }}
+                  </span>
+                </div>
+              </td>
 
-        <div v-else class="flex justify-center">{{ t('users.no_users') }}</div>
+              <td class="py-2 px-7">
+                {{ user.email }}
+              </td>
+
+              <td class="py-2 px-7">
+                {{ getCreatedDate(user.createdOn) }}
+                <div class="text-gray-400 text-tiny">
+                  {{ getCreatedTime(user.createdOn) }}
+                </div>
```

Comment:
```suggestion
                <span class="text-gray-400 text-tiny">
                  {{ getCreatedTime(user.createdOn) }}
                </span>
```

Will a `span` work in this case? The table cell contents only need to be an inline element, right?

Reply:
The created time should be in a new line. Replacing it with a span will result in both the date and time to be in the same line.